browser-test/src/support/applicant_questions.ts

```diff
@@ -82,7 +82,7 @@ export class ApplicantQuestions {
 
   async submitFromReviewPage(programName: string) {
     // Assert that we're on the review page.
-    expect(await this.page.innerText('h1')).toContain('Application summary');
+    expect(await this.page.innerText('h1')).toContain('Program review');

```

Comment:
question: I see "Program preview" rather than "review" in the screenshot on the PR description.  Is this testing the last page rather than the first page version?

Reply:
Nvm - saw the titles in messsages